analysis/import/GitLogParser/README.md

```diff
@@ -32,35 +32,66 @@ The names of authors are saved when the --add-author flag is set.
 
 ## Usage
 
-### Creating the repository log for metric generation
+### Creating required files on the fly | repo-scan
+
+See `ccsh gitlogparser repo-scan -h` for help. Standard usage:
+
+> `ccsh gitlogparser repo-scan --repo-path <path>`
+
+With the sub command `repo-scan`, the git log and the file name list are created automatically either from the
+current working directory or from the directory at `repo-path`.
```

Comment:
The user might not know that we are going to create a git log and a file name list. So I would change this to:
"With the sub command `repo-scan` you can parse a local git repository on your disk. During scanning a git log of the repository in the current working directory (or from the directory specified by `repo-path`) is created and parsed automatically. Furthermore, the parser creates a file name list of files that are tracked by git automatically which is needed for the parsing process.